src/engine/Sim3D.ts

```diff
@@ -337,4 +343,82 @@ export class Sim3D {
     const handle = new RobotHandle(robot, robotRoot);
     return handle;
   }
+
+  isDebugMode(): boolean {
+    return this.debugMode;
+  }
+
+  setDebugFlag(debug: boolean): void {
+    this.debugMode = debug;
+  }
+
+  renderWireframes(): void {
+    const debugScene = new THREE.Scene();
```

Comment:
Yeah I'm going to experiment with this, this PR is a fairly hacky implementation at the moment and I do want to do some house cleaning. The reason I've done it like this is so that we don't pay any overhead when not in debug mode, but there's definitely neater ways of achieving this